### ISSUE TITLE
[Meta] Implement Openverse redesign

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse-frontend#302]
Openverse is live with transitional styles at wp.org/openverse. Openverse will be redesigned with a new brand, color palette, and UI designed by @panchovm. 

Concurrently with this redesign, we will begin using our new approach to components. That is described in detail [in this discussion](https://github.com/WordPress/openverse-frontend/discussions/283#discussioncomment-1449819). Essentially, we will be building a new small framework of components  to power the Openverse front-end. 

## Resources

- Public Figma File (blocked by: WordPress/openverse#91)

## Tasks

- [ ] Outline component requirements and structure of the header component (blocked by WordPress/openverse#92)

**Comment by zackkrida** [comment]
Closing in favor of #364
